src/main/java/com/puppycrawl/tools/checkstyle/checks/SuppressWarningsHolder.java

```diff
@@ -87,11 +87,11 @@
      * @return the default alias for the given check
      */
     public static String getDefaultAlias(String sourceName) {
-        final int startIndex = sourceName.lastIndexOf('.') + 1;
```

Comment:
if we want `final` variables closer to where we are used, should we turn this on for our own check?
http://checkstyle.sourceforge.net/config_coding.html#VariableDeclarationUsageDistance
`ignoreFinal` property.